ember_debug/utils/ember/runloop.js

```diff
@@ -1,21 +1,21 @@
-import Ember from '../ember';
+import Ember, { Runloop as EmberRunloop } from '../ember';
 import * as runloop from './own-runloop';
 
+// it could happen that runloop is available but _backburner is not exported (dead code)
+// then we need to use our own.
 let module = runloop;
 let _backburner = runloop._backburner;
 
 const keys = ['cancel', 'debounce', 'join', 'later', 'scheduleOnce'];
 
-try {
-  module = requireModule('@ember/runloop');
-  // it could happen that runloop is available but _backburner is not exported (dead code)
-  // then we need to use our own
-  _backburner = module._backburner;
-} catch {
-  // eslint-disable-next-line ember/new-module-imports
-  _backburner = Ember?.run?.backburner || module._backburner;
+if (EmberRunloop) {
+  module = EmberRunloop;
+  _backburner = EmberRunloop._backburner;
+} else {
   // eslint-disable-next-line ember/new-module-imports
   module = Ember?.run || module;
+  // eslint-disable-next-line ember/new-module-imports
+  _backburner = Ember?.run?.backburner || _backburner;
```

Comment:
Just reverted those blocks so `module` is assigned before `_backburner`, because it's the order chosen at the top of the file so it eases the reading.